tools/docker/Dockerfile

```diff
@@ -40,6 +40,7 @@ RUN apt-get -qq update && \
     lib32z1 \
     libcanberra-gtk-module \
     libcoap2-bin \
+    libgl1-mesa-glx \
```

Comment:
Please add a comment on line 17 that the library is required by Simplicity Commander so it's clear if it can be removed in the future.

Reply:
Done. :)